Bug in getting-started.md undefined: semconv

### Description

I was following the steps in the getting-started.md to run the fib trace example.  
When I get to the $ go run . I get the following errors:

 go run .
# fib
./main.go:22:4: undefined: semconv
./main.go:23:4: undefined: semconv
./main.go:24:4: undefined: semconv

I then added the following import to main.go

"go.opentelemetry.io/otel/semconv/v1.10.0"

And then everything worked.   
Should this import be added to the example or did I do something wrong that I needed that?


### Environment

- OS: mac OS 11.6
- Architecture: [e.g. x86, i386] 
- Go Version: [e.g. 1.15] go version go1.17.6 darwin/amd64
- 
- opentelemetry-go version: [e.g. v0.14.0, 3c7face]
- bradtopol@Brads-MBP fib % go get go.opentelemetry.io/otel/sdk \
         go.opentelemetry.io/otel/exporters/stdout/stdouttrace
go: downloading go.opentelemetry.io/otel/sdk v1.7.0
go: downloading go.opentelemetry.io/otel v1.7.0
go: downloading go.opentelemetry.io/otel/exporters/stdout v0.20.0
go: downloading go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.7.0
go: downloading go.opentelemetry.io/otel/trace v1.7.0
go get: upgraded go.opentelemetry.io/otel v1.6.3 => v1.7.0
go get: added go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.7.0
go get: added go.opentelemetry.io/otel/sdk v1.7.0
go get: upgraded go.opentelemetry.io/otel/trace v1.6.3 => v1.7.0
go get: added golang.org/x/sys v0.0.0-20210423185535-09eb48e85fd7

### Steps To Reproduce
See above

### Expected behavior

I expected it to work without me having to add an extra import

If the example needs the extra import I am happy to submit a PR for this.

--- a/website_docs/getting-started.md
+++ b/website_docs/getting-started.md
@@ -284,6 +284,7 @@ import (
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/exporters/stdout/stdouttrace"
 	"go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.10.0"
 )
 ```
 

